### PR TITLE
clean the pytests such that my releaseScript runs successfully

### PR DESCRIPTION
**Pull request by SteffenBrinckmann**

### CODE REVIEW

**Comment by SteffenBrinckmann** [comment]
@jmurugan-fzj Can you review this one, too?

File: pasta_eln/Extractors/extractor_json.py [hunk 1]
```diff
@@ -16,13 +17,17 @@ def use(filePath, recipe='', saveFileName=None):
   """
   # Extractor for fancy instrument
   content = ''
-  with open(filePath,'r', encoding='utf-8') as jsonFile:
-    jsonContent = jsonFile.read()
-    content = json.loads(jsonContent)
-    if not isinstance(content, dict):
-      content= {'content': json.loads(jsonContent)}
+  fileSize = os.stat(filePath).st_size / (1024*1024)  #in MB
```
Comment (jmurugan-fzj) [inline pasta_eln/Extractors/extractor_json.py:20]: Why do we limit the size of the json? Anything above 0.3 Mb creates some performance problem?

File: tests/test_03_3Projects_ExportImport3.py [hunk 2]
```diff
@@ -27,6 +27,122 @@ def test_main(self):
     """
     main function
     """
+    ###### BEGIN testELNFile: copy into all export tests in this folder   ######
+    def testELNFile(fileName:str, verbose:bool) -> None:
```
Comment (jmurugan-fzj) [inline tests/test_03_3Projects_ExportImport3.py:31]: Why do we duplicate the same function in both classes, is there any specific reason to do so? I would go for the previous implementation itself, importing testELNFile from tools!

**Comment by jmurugan-fzj** [comment]
> @jmurugan-fzj Can you review this one, too?

@SteffenBrinckmann Only have two comments, please see them..

File: pasta_eln/Extractors/extractor_json.py (hunk 1 above)
Comment (SteffenBrinckmann) [reply pasta_eln/Extractors/extractor_json.py:20]: Yes, indeed. If people come with super large json, the performance goes down and nobody will look through those multi-MB json anyhow.

File: tests/test_03_3Projects_ExportImport3.py (hunk 2 above)
Comment (SteffenBrinckmann) [reply tests/test_03_3Projects_ExportImport3.py:31]: @jmurugan-fzj I tried that but then I get a fixtures error. I tried all morning to understand / wrap my head around fixutures. I asked chatGPT. But nobody could help.

**Pull request merged**